external/webserver/http_client.c

```diff
@@ -1257,13 +1257,19 @@ int http_send_response(struct http_client_t *client, int status, const char *bod
 {
 	const char* status_message = (status == 200) ? "OK" : body;
 
-	int body_len = strlen(body);
+	int body_len = 0;
+	if (body) {
+		body_len = strlen(body);
+	}
```

Comment:
`body` and `body_len` are given to `http_send_response_helper`. But it checks the `body` with null always and `body_len` is used in body conditional. Is this really necessary?

Reply:
Use of strlen function with body param value as NULL will give error. So Null check is added to avoid the issue.